hash/gbcolor.xml

```diff
@@ -25953,12 +27056,12 @@ These were produced between 2000 and 2001 by Rocket Games, run by Datel Design
 		</part>
 	</software>
 
-<!-- This is a title hack of an undumped Sintax game: 真三國無雙 (Zhen San Guo Wu Shuang) whose 'official' english title is True Three Kingdoms -->
 	<software name="cbzz">
-		<!-- Alt. Title: 吞食天地 赤壁之戰 (Devouring of Heaven and Earth: The Battle of Red Cliffs) -->
+	<!-- This is a title hack of an undumped Sintax game: 真三國無雙 (Zhen San Guo Wu Shuang) whose 'official' english title is True Three Kingdoms -->
 		<description>Tun Shi Tian Di - Chi Bi Zhi Zhan (Chi)</description>
 		<year>20??</year>
 		<publisher>&lt;unknown&gt;</publisher>
+		<info name="alt_title" value="吞食天地 赤壁之戰 (Devouring of Heaven and Earth: The Battle of Red Cliffs)"/>
```

Comment:
Please don’t put translations in `alt_title` info – they’re too subjective.  Only use official titles.

Reply:
I didn't touch any pirate stuff.

Reply:
Then why does it show in the diff?

Reply:
Because those alt names were in the softlist as comments instead of "info" lines.

Reply:
I've just removed all those translations.